kubernetes/Makefile

```diff
@@ -79,6 +108,23 @@ build/vars-merged-with-secrets.yml: var.yml var-$(CLUSTER).yml $(CONCENT_SECRET_
 build/image-version.yml:
 	echo "image_version: $(IMAGE_VERSION)" >> build/image-version.yml
```

Comment:
While we're at it, you could use `>> "$@"` here too.

Reply:
Done.